programs/admin.py

```
@@ -598,6 +600,20 @@ class CategoryIconNameAdmin(SecureAdmin):
     search_fields = ("name",)
 
 
+class IconAdmin(SecureAdmin):
+    list_display = ["name", "lucide_name", "description"]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)